tox.ini

```diff
@@ -18,4 +18,4 @@ commands =
     scrapy1x: pip install scrapy>=1.0,<2.0
     scrapyrel: pip install scrapy
     scrapydev: pip install https://github.com/scrapy/scrapy/archive/master.zip
-    {posargs:coverage run -m py.test}
+    {posargs:coverage run -m pytest }
```

Comment:
Please change `posargs:coverage run -m pytest`  to `posargs:coverage run -m pytest --ignore=setup.py`.